### DISCUSSION TITLE
Is this integration getting updates via polling method or push/pull method?

### DISCUSSION THREAD

**Discussion by Majestic7979** [sanjoyg/dirigera_platform#102] [General]
Hi, firstly I hope that you the developer is doing well and this finds you in good spirits and thanks for writing this integration!

I wanted to know if this is using polling method because it seems that my devices when connected to this react very slowly in comparison to if they were joined to Zigbee2MQTT, the problem is that Zigbee2MTT is buggy for many IKEA devices namely FYRTUR blinds and have caused them to over-run their limit points and cause damage to the blinds by rolling them in the u direction. It also fails to stop at the top and keeps trying to wind them tight non-stop which concerns me because it could burn the motor. Generally, it is useless to apportion blame in this situation, not sure if IKEA has bad firmware in the motor, or if ZHA/Z2M have bad programming/bugs. Either way, I found the Dirigera handles the blinds more gracefully and safe... so I am relegated to using it.

But then, I found that the pairing process is much more reliable and I will soon be moving my 20+ PARASOLL sensors to Dirigera, because I am finding their batttery drains quickly, like within 2-3 weeks after replacing full fully charged LADDA batteries per IKEA's recommendation and that's simply not feasible for me. Replacing is a major chore!

It would be great if we could get instant updates rather than using polling, or at least control how often the polling occurs (I understand if it's there for rate limiting/avoiding network congestion). I just find it way too slow :-( So the curtain may be reporting it's at 50% position while it's moving up, when in fact it may be already 80% open but the polling is so slow it's not sending the position quickly enough to Home Assistant which can cause automation issues...

So, is this integration, or my setup? Any ideas please? :)

**Comment by sanjoyg** [comment]
Thank you for asking :-) am doing well. The integration does both, I just released a beta release that should get faster updates. I would request you to do the following
1. Update to the beta release 2.5.1
2. Enable Debugging
3. Try the blinds and parasoll

Do paste the debug log and if u see any errors. When the blinds are updated or parasoll sensors change state an hub event is generated which updates the status in the integration. So look out for hub events in the log and the code that follows

**Reply by Majestic7979** [reply]
I'm glad to know you're well :-)

Thanks for your response, I'll try it out and report!

God bless/or stay well according to your preferred belief 😁
